GSoC2019/README.md

```diff
@@ -26,7 +26,11 @@ Everything is inside the src folder.
  
 ## Local deploy
 
-Just access the index.html file in Firefox browser (in Chrome there is a restriction that requires accessing files only through HTTP or HTTPS).
+The files can be hosted locally by running the following command in the `GSoC2019/src` folder.
+```
+python -m http.server 4000
+```
+The visualization can be found by navigating to http://localhost:4000/ in the browser.
 
 ## Visualization in production

```

Comment:
Could you change this title please, as these visualizations were developed during the program, but the product is the one you link to in the README file (the one in the root directory).

Reply:
OK. Should I rename it as `The Linked Commons`?